webapp/app/dataExport/utils/panEuropean.js

```diff
@@ -0,0 +1,4 @@
+import { tableMappings } from '@webapp/app/dataExport/utils/panEuropeanMappings'
+
+export const getPanEuropeanTableMapping = (section) =>
```

Comment:
This function is used to get the correct table mapping for server/traditionalTable/mappings/panEuropean/index.js